src/db/migrations/20190723163335-create-user.js

```diff
@@ -0,0 +1,29 @@
+
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
```

Comment:
Please use async-await and try/catch. i.e 
```suggestion
  async up(queryInterface, Sequelize) { 
    try {
      queryInterface.createTable('Users', {...
    } catch (error) {
      console.log(error);
    }
```

This will help us to easily catch errors that occur during migration. Please do the same thing for `down`.